app/components/on-site-notifications/notification-card.js

```diff
@@ -295,6 +303,98 @@ export default Component.extend(
 			});
 		},
 
+		getMessageWallPostBody(model) {
+			let wallOwner = model.get('metadata') && model.get('metadata.wallOwnerName');
```

Comment:
`model.get` is null-safe, so you can use `model.get('metadata.wallOwnerName')` by itself.